_includes/geek-details/datastore-etcd.md

```diff
@@ -0,0 +1,15 @@
+Calico stores the operational and configuration state of your cluster in a central datastore. If the datastore is unavailable your Calico network continues operating, but cannot be updated (no new pods can be networked, no policy changes can be applied, etc.).
```

Comment:
Is this geek-detail used anywhere? I only see datastore-kubernetes being used on the GKE page

Reply:
It's not used by GKE page, but was easy to author while doing the kdd one.  All of the install pages will have geek details once we are finished.  In addition we anticipate using geek details on some of the how-to or reference pages in a "this requires the following" type of style.